python_ta/.pylintrc

```diff
@@ -101,3 +101,24 @@ argument-rgx = (([a-z][a-z0-9_]{0,30})|(_[a-z0-9_]*))$
 class-rgx    = (([A-Z][a-zA-Z0-9]{0,30})|(_[A-Z][a-zA-Z0-9]*))$
 method-rgx   = (([a-z][a-z0-9_]{0,30})|(_[a-z0-9_]*))$
 class-attribute-rgx = ([A-Za-z_][A-Za-z0-9_]{0,30}|(__.*__))$
+
+[PYCODESTYLE]
+# Ignored pycodestyle checks
+pycodestyle-ignored-tests =
```

Comment:
Rename this option to `pycodestyle-ignore` (it's more consistent with how pycodestyle names it).